### PR TITLE
chore: Mock keycard option added in start AUT

### PR DESCRIPTION
**Pull request by StateOf-Vlado**
#260
Test run: https://ci.status.im/job/status-desktop/job/e2e/job/manual/795/

Example:
```
from constants import aut_options
@pytest.mark.parametrize('options', [aut_options.MOCK_KEYCARD])
def test_create_keycard_account_with_new_seed_phrase(main_screen: MainWindow):
    pass
```

### CODE REVIEW

**Comment by anastasiyaig** [comment]
i am seeing the attached run is failed. Please check it out @StateOf-Vlado . Also, next time please run tests before asking for review to make sure they are good and attach the screenshot of successful (green) run.

It will be changed soon when the launch of the tests will be implemented for PRs in this repository.

File: constants/aut_options.py [hunk 1]
```diff
@@ -0,0 +1 @@
+MOCK_KEYCARD = '--test-mode=1'
```
Comment (Valentina1133) [inline constants/aut_options.py:1]: Extra line needed

**Comment by StateOf-Vlado** [comment]
> i am seeing the attached run is failed. Please check it out @StateOf-Vlado . Also, next time please run tests before asking for review to make sure they are good and attach the screenshot of successful (green) run.
> 
> It will be changed soon when the launch of the tests will be implemented for PRs in this repository.

Fixed

File: constants/aut_options.py (hunk 1 above)
Comment (StateOf-Vlado) [reply constants/aut_options.py:1]: fixed

**Review by Valentina1133 — APPROVED** [review]

**Pull request merged**